cl/search/fixtures/test_objects_query_counts.json

```diff
@@ -418,7 +423,8 @@
          "date_created":"2015-08-15T14:10:56.801Z",
          "html_lawbox":"",
          "per_curiam":false,
-         "type":"010combined"
+         "type":"010combined",
+         "ordering_key": 4
```

Comment:
im not sure I understand what is happening here.  

How do we have ordering keys for combined opinions - shouldn't they be null 

and also why are they also alternatively not 1